spec/jobs/tts_job_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe TtsJob, type: :job do
+  include ActiveJob::TestHelper
+
+  subject(:job) { described_class.perform_later }
+  let(:word) { create(:noun, with_tts: with_tts) }
+
+  after do
+    clear_enqueued_jobs
+    clear_performed_jobs
+  end
+
+  context "when with_tts is false" do
+    let(:with_tts) { false }
+
+    it "doesn't generate the audio" do
+      expect(TtsGenerator).not_to receive(:call)
+      perform_enqueued_jobs { job }
+    end
+  end
+
+  context "when with_tts is true" do
+    let(:with_tts) { true }
+
+    it "doesn't generate the audio" do
```

Comment:
We don't have a "real" test with a mocked HTTP response which actually tests the code, but I guess that's too much effort for now and it probably works anyways :slightly_smiling_face: